disk/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "maintainer": "help@datadoghq.com",
-  "manifest_version": "1.0.0",
+  "manifest_version": "1.2.0",
```

Comment:
this is the version of the manifest, no need to bump it if you don't change the manifest format